### DISCUSSION TITLE
Qt6 patch

### DISCUSSION THREAD

**Discussion by edpil02** [falkTX/Carla#1913] [General]
I made a qt6  version of Carla.
Need help to improve my patch.
[carla.zip](https://github.com/user-attachments/files/16539512/carla.zip)

**Comment by edpil02** [comment]
![carla-qt6](https://github.com/user-attachments/assets/73f2825e-1769-4229-8e4e-c62937e64302)

**Comment by falkTX** [comment]
Thanks for the patch! this will help with the transition.

But I am not happy about updating to Qt6 right now, it requires updating the codebase to C++17 and drops support for many older OSes.
Qt5 still works fine so it will remain the default in Carla for the time being.

But I will make it so Qt6 build is possible.

**Comment by ohquait** [comment]
just aquestion:
will qt6 be stable enough?
and if possible split by build-process with hint for differs in qt5 and qt6?

**Reply by falkTX** [reply]
should be the same as with qt6, but initially will need some testing.

I can add some "make features" print for qt5 vs qt6 sure

**Comment by ohquait** [comment]
would be nice,
Thank YOU for the works

Am Mo., 9. Dez. 2024 um 09:09 Uhr schrieb Filipe Coelho <
***@***.***>:

> should be the same as with qt6, but initially will need some testing.
>
> I can add some "make features" print for qt5 vs qt6 sure
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/falkTX/Carla/discussions/1913#discussioncomment-11505664>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AP5C6CYPXJ2JVOUIEU2SOU32EVM4ZAVCNFSM6AAAAABMF434SWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNJQGU3DMNA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
also on ***@***.***
